packages/web-components/@sfx/sayt/stories/index.ts

```diff
@@ -1,25 +1,48 @@
 import { storiesOf } from '@storybook/html';
 import { withKnobs, text, boolean } from '@storybook/addon-knobs';
+
 import '../src/index.ts';
+import { getDisplayCode } from '../../../../../.storybook/common';
```

Comment:
Not sure if it matters, but should we keep the import types grouped together/or in a certain order? (single imports vs. files)

Reply:
The order I've been suggesting is non-local imports (in some order that makes sense; alphabetical if you can't think of one) followed by local imports. For tests: non-local imports, test utilities, the class under test, and then other supporting imports. All imports should be in one block (i.e. no spaces between imports).

Reply:
The import order is normally based on imports being 1) built-in, 2) third-party, and 3) project files. I am switching the order slightly for prettiness, though.